web/Views/Shared/_Layout.cshtml

```diff
@@ -85,7 +85,7 @@
 
 
 <main role="main">
-    <div class="container mt-3 pb-4">
+    <div class="container@(TempData["Wide"] is true ? "-fluid" : "") mt-3 pb-4">

```

Comment:
I think possibly we should change this from just `"Wide"`. Is there a more descriptive name we could use?

Reply:
Changed to `"FullWidthPage"`